hangman.py

```diff
@@ -0,0 +1,60 @@
+import random
+
+
+"""
+The hangman game the program chooses a random word from list and prompts the user to guess letters in word.
+It keeps track of user's guesses and fills in
+the blanks for correctly guessed letters until the user guesses the word or runs out of guesses    
+    
+    
+"""
+
+
+class Game:
+    def __init__(self):
+        with open("wordsforhangman.txt", "r") as f:
+            self.data = f.readlines()
+            self.line = random.choice(self.data)
+            self.number_of_lifes = 12
```

Comment:
mozna też krócej - `self.lifes`. Uncle bob mówi, żeby dobierać dokładność opisu zmiennych do scope/kontekstu. Jeśli wiadomo co jest co, to najlepiej jest wybierać jak najkrótsze nazwy - szczególnie taka konwencja panuje w pythonie.